lib/morphix.ex

```diff
@@ -56,8 +56,16 @@ defmodule Morphix do
 
   ```
 
+  `equaliform?/2` compares two ordered or unordered lists and returns `true` if they are equal.
+
+  ### Example:
+  iex> Morphix.equaliform?([1, 2, 3], [2, 3, 1])
+  true
+
```

Comment:
maybe add a test for the error case and a more complex example, does this work for a list with complicated nested values? I assume it does but what about values that might sort in different ways... how do nested maps sort?

Reply:
I added a nested list and a map element with a nested map to the example. Let me know if there's anything else I should add to make it better. Thanks!

Reply:
```
iex(12)> Morphix.equaliform?([%{a: [1,2,3]}], [%{a: [1,2,3]}])
true
iex(13)> Morphix.equaliform?([%{a: [1,2,3]}], [%{a: [3,2,1]}])
false
```

Because `[1,2,3]` is considered to have unordered equality to `[3,2,1]`, I think that we would consider these two lists to have unordered equality.

Reply:
I had more comments and they appear to have disappeared. This is bad, because they were pretty extensive.

Reply:
I received an email with the notes, re: equalify? vs equaliform?

Reply:
I have fixed the nested list in a map issue and looking into equalify? vs equaliform?.